File: Exemplos/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/TERMICA/Commands/ImprimeImagem.ts
import { TermicaCommand } from "../TermicaCommand";

export class ImprimeImagem extends TermicaCommand {
  readonly path: string;

  constructor(path: string) {
    super("ImprimeImagem");
    this.path = path;
  }

  functionParameters(): object {
    return { path: this.path };
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Termica/ImprimeImagem.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Termica
{
    class ImprimeImagem : TermicaCommand
    {
        readonly string path;

        public ImprimeImagem(string path) : base("ImprimeImagem")
        {
            this.path = path;
        }

        protected override string FunctionParameters()
        {
            return "\"path\"" + ":" + "\"" + path + "\"";
        }
    }
}